bundler/helpers/v2/lib/functions/dependency_source.rb

```diff
@@ -0,0 +1,86 @@
+module Functions
```

Comment:
Could also copy these native helper specs over for v2 https://github.com/dependabot/dependabot-core/blob/main/bundler/helpers/v1/spec/functions/dependency_source_spec.rb